index.pug

```diff
@@ -377,6 +391,64 @@ html(lang='en')
+
+            img(style='max-width: 100%;', src='img/localization_process.png', alt='localization-process')
+
+    section#platform.platform
+      .container
+        .row
+          .col-md-12
+            h2 What can our localization platform do?
+            p You and your team can manage the following types on our localization platform. We help you start your language localisation on all apps, games and documents. A localization-management-tool will improve the quality of your process management by store all your files in one place during the development.
+
+            .col-md-4.text-center
+              a(href='#')
+                img(style='max-width: 100%;', src='img/website_localization.png', alt='website-localization.png')
+              h3 Website
+              p Use our tool for web applications of any kind. This can be frontend, backend, landingpage or blogs - choose your fit. 
+
+            .col-md-4.text-center
+              a(href='#')
+                img(style='max-width: 100%;', src='img/app_localization.png', alt='app-localization')
+              h3 Mobile App
+              p Take your mobile app to next level. Manage your mobile platform language easily and eliminate unnecessary communication in your team.
+
+            .col-md-4.text-center
+              a(href='/customers.html')
+                img(style='max-width: 100%;', src='img/game_localization.png', alt='game-localization')
+              h3 Video game
+              p Automate your game translation management with Locize and let your team save a lot of time during development process.
+
+            .col-md-4.text-center
+              a(href='#')
+                img(style='max-width: 100%;', src='img/platform_localization.png', alt='platform_localization')
+              h3 Platforms
+              p We helped platforms to include multiple languages into their software solutions, that they can provide a more advanced service for their users and suppliers.  
+
+            .col-md-4.text-center
+              a(href='/how-it-works.html')
+                img(style='max-width: 100%;', src='img/website_builder.png', alt='website_builder')
+              h3 Website builders
+              p We have built a custom script for Website Builders where flexible translation options are available. This can be used with Wordpress, Shopify and other Website Builders. 
+
+            .col-md-4.text-center
+              a(href='#')
+                img(style='max-width: 100%;', src='img/documents.png', alt='documents')
+              h3 Documents
+              p Manage your language-localisation in Word, Excel, JSON and static HTML files. There are no limitations. <a href="/customers.html" title="">Read full case study</a>. 
```

Comment:
Documents is probably more of a localistars topic....
there is no possibility in locize to handle word or html files directly.

Reply:
ok I will remove this